### ISSUE TITLE
[Survival Mode] All component items turned into plants

### ISSUE THREAD

**Issue by surajnairpv** [tsunamayo/Starship-EVO#5546]
### Short Description of the issue:
I wanted to remove the assembler which was stuck (earlier mentioned issue with assembler stopping construction once you stack up multiple to do builds #5536 )
Before taking it out I took all the constructed components into my bag pack, after replacing the Assembler. My bag pack components have turned into plants.
...

### Steps to reproduce the issue:
Build an assembler 
stack up components into the bag pack
remove the assembler
create a new one
check the components in the bag pack

...

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

![Starship_EVO_qS4bFyfxkX](https://github.com/tsunamayo/Starship-EVO/assets/93534629/dcd03e30-64f9-4a23-aa88-ef3ca782ba99)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/14319649/Player.log)

**Comment by tsunamayo** [comment]
@surajnairpv Thanks, it is clear we have a corruption of some kind happening, I need to find a way to reproduce. Was the assembler producing something when you deleted it? Thanks

**Comment by surajnairpv** [comment]
No it wasn't, it only had some components in its inventory. Which I had taken to reuse later.
Once the assembler was clear that's when I deleted it.

**Comment by tsunamayo** [comment]
@surajnairpv Like I feared I cant reproduce. Did you had a furnace / h2 farm working at the same time?

**Comment by surajnairpv** [comment]
Yes the furnace was working on the side had kept some gold for smelting ingots.
